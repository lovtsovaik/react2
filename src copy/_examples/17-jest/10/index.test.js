global.Date = jest.fn();

class User {
    constructor(name) {
        this.name = name;
        this.created = new Date();
    }
}

test('«User» constructor should match its snapshot inline', () => {
    expect(new User('Jack')).toMatchSnapshot();
});
