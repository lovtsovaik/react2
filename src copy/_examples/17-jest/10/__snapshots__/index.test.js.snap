// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`«User» constructor should match its snapshot inline 1`] = `
User {
  "created": 2021-03-12T18:46:00.253Z,
  "name": "Jack",
}
`;
